OpenRA.Mods.D2/FileFormats/WsaReader.cs

```diff
@@ -57,17 +57,21 @@ public void Read(Stream stream, out TypeDictionary metadata)
 
 			wsaLoader.TryParseSpriteWithPrevFrame(stream, prev, out videoFrames, out metadata);
 
-			if (frames == null) {
+			if (frames == null)
+			{
 				frames = videoFrames;
 
-				if (Length > 0) {
+				if (Length > 0)
+				{
 					var frame = frames[0];
 					width = frame.Size.Width;
 					height = frame.Size.Height;
 				}
 			}
 			else
+			{
```

Comment:
Fwiw, the convention inside the codebase of OpenRA/OpenRA is to omit braces around single-line statements.